model-api/src/commonMain/kotlin/org/modelix/model/api/ITree.kt

```diff
@@ -254,6 +254,12 @@ interface ITree {
      */
     fun deleteNodes(nodeIds: LongArray): ITree
 
+    /**
+     * @param nodeId id of the node
+     * @param concept the concept ref of the new concept for the node
```

Comment:
Why is it nullable? What's the meaning in case of `null`?

Reply:
Concepts are optional in the model-datastructure. I added a hint.